packages/scandipwa/src/component/Loader/Loader.style.scss

```diff
@@ -40,3 +40,13 @@
         }
     }
 }
+
+#root {
```

Comment:
You don't need this extra `#root` nesting, since all components are rendered within.